jobs/picard.py

```diff
@@ -65,3 +65,178 @@ def markdup(
             )
 
     return j
+
+
+def picard_collect_metrics(
+    b,
+    cram_path: CramPath,
+    out_alignment_summary_metrics_path: Path,
+    out_base_distribution_by_cycle_metrics_path: Path,
+    out_insert_size_metrics_path: Path,
+    out_quality_by_cycle_metrics_path: Path,
+    out_quality_yield_metrics_path: Path,
+    job_attrs: dict | None = None,
+    overwrite: bool = False,
+) -> Job | None:
+    """
+    Run picard CollectMultipleMetrics metrics for sample QC.
+    Based on https://github.com/broadinstitute/warp/blob/master/tasks/broad/Qc.wdl#L141
+    """
+    if can_reuse(
+        [
+            out_alignment_summary_metrics_path,
+            out_base_distribution_by_cycle_metrics_path,
+            out_insert_size_metrics_path,
+            out_quality_by_cycle_metrics_path,
+            out_quality_yield_metrics_path,
+        ],
+        overwrite,
+    ):
+        return None
+
+    job_attrs = (job_attrs or {}) | {'tool': 'picard_CollectMultipleMetrics'}
+    j = b.new_job('Picard CollectMultipleMetrics', job_attrs)
+    j.image(image_path('picard'))
+    res = STANDARD.request_resources(mem_gb=7)
```

Comment:
No functional comment here, but 7gb seems like an odd reservation!

Reply:
That's just directly borrowed from WARP :)

Reply:
But good point, to make it clearer, I'll modify to request the minimal instance chunk instead with `res = STANDARD.request_resources(ncpu=2)`, it would correspond to ~7G